include/drivers/i2c/sun6i-a31-i2c.h

```diff
@@ -8,6 +8,9 @@
 
 #include <i2c.h>
 
+#define I2C_NUM_PINS 2
```

Comment:
If these are generic to all I²C controllers, they should go in `i2c.h`

Reply:
Done.